npm/cmd/start.go

```diff
@@ -29,6 +31,12 @@ import (
 	"k8s.io/utils/exec"
 )
 
+var npmV2DataplaneCfg = &dataplane.Config{
+	PolicyMode:       "",
```

Comment:
PolicyMode is what we use in PolicyManagerCfg i think ? we can safly remove this PolicyMode from DpConfig and use it from PolicyManagerCfg ?

Reply:
Also we can just name it npmDataplaneCfg..

Reply:
Eventually, I think we should rename Mode to PolicyMode since Dataplane.Config inherits from PolicyManagerCfg, but we'll leave that for another PR